build/run-restore.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+set -o xtrace
+
+DATADIR=${DATADIR:-/var/lib/mysql}
+PARALLEL=$(grep -c processor /proc/cpuinfo)
+XBCLOUD_ARGS="--curl-retriable-errors=7 --parallel=${PARALLEL}"
+INSECURE_ARG=""
+if [ -n "$VERIFY_TLS" ] && [[ $VERIFY_TLS == "false" ]]; then
+	INSECURE_ARG="--insecure"
+fi
+
+run_s3() {
+	xbcloud get ${XBCLOUD_ARGS} ${INSECURE_ARG} ${BACKUP_DEST} --storage=s3 --s3-bucket=${S3_BUCKET}
+}
+
+run_gcs() {
+	xbcloud get ${XBCLOUD_ARGS} ${INSECURE_ARG} ${BACKUP_DEST} --storage=google --google-bucket="${GCS_BUCKET}"
+}
+
+run_azure() {
+	xbcloud get ${XBCLOUD_ARGS} ${INSECURE_ARG} ${BACKUP_DEST} --storage=azure
+}
+
+extract() {
+	local targetdir=$1
+
+	xbstream -xv -C ${targetdir} --parallel=${PARALLEL}
+}
+
+main() {
+	echo "Starting restore ${RESTORE_NAME}"
+	echo "Restoring to backup ${BACKUP_NAME}: ${BACKUP_DEST}"
+
+	rm -rf ${DATADIR}/*
+	tmpdir=$(mktemp --directory)
```

Comment:
```suggestion
	tmpdir=$(mktemp --directory /datadir/pxc_sst_XXXX)
```

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/80#discussion_r902573430

Reply:
ok, but please do not use tmp, please use` /var/lib/mysql` local storage can be small for the backup